demo/server/src/auth/local.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express';
+
+export function loginRequired(req: express.Request, res: express.Response, next: Function) {
```

Comment:
Is this method going to be the same as the higher level `keycloak.protect()` to protect routes and check auth/roles, or is it going to be similar to the `login-required` option in the keycloak init method. Just the naming is causing a bit of confusion.